docs/contributing.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+Contributions are very welcome, just open an issue or PR if you see something to improve! See our other [communication channels](../README.md#getting-involved) above to coordinate with other contributors!
+
+Please note that all contributions happen under the MIT license as described at the bottom of this page.
+
+## Getting involved
+
+If you want to learn more about the general idea go to [fedimint.org](https://fedimint.org/), beyond a high level explanation you can also find links to talks and blog posts at the bottom of the page.
+
+Visit our 
```

Comment:
This is a dangling reference yet I still really want to visit wherever it goes! 😎

Reply:
:eyes: Fixed.